src/renderer/store/modules/utils.js

```diff
@@ -27,6 +27,7 @@ const state = {
     movies: null
   },
   cachedPlaylist: null,
+  deArrowCache: [],
```

Comment:
I think it's possible to store one object per video ID

See https://github.com/FreeTubeApp/FreeTube/pull/3668 for what's done for cache per channel
```suggestion
  deArrowCache: {},
```